src/particle_filter.h

```diff
@@ -81,8 +81,7 @@ class ParticleFilter {
 	 * updateWeights Updates the weights for each particle based on the likelihood of the 
 	 *   observed measurements. 
 	 * @param sensor_range Range [m] of sensor
-	 * @param std_landmark[] Array of dimension 2 [standard deviation of range [m],
-	 *   standard deviation of bearing [rad]]
+	 * @param std_landmark[] Array of dimension 2 [Landmark measurement uncertainty [x [m], y [m]]]
```

Comment:
A common approach to passing complex data in C++ is using a structure with descriptive member names, rather than an opaque array or vector. Thus the code will be efficient and descriptive by itself, without having to rely on comments. In this case the structure should be defined as:
```
class ParticleFilter {
public:
  struct StdLandmark {
    double x;
    double y;
    StdLandmark() : x(0), y(0) { }
    StdLandmark(double in_x, double in_y) : x(in_x), y(in_y) { }
  };
  void updateWeights(double sensor_range, StdLandmark std_landmark, std::vector<LandmarkObs> observations,
                                     Map map_landmarks);
```